src/storage/src/source/healthcheck.rs

```diff
@@ -228,25 +255,12 @@ impl Healthchecker {
         status_update: &SourceStatusUpdate,
```

Comment:
can't select above the changes but:

almost certain this code won't live that much longer -- but just in case, perhaps add a function on `SourceStatus` to correctly preserve the error message instead of just `SourceStatus::try_from(String)` which just uses an empty error message

Reply:
Noted: I also suspect this is close to death, but if we can't get rid of it I will do as you describe!